src/js/lib/client/src/preload.js

```diff
@@ -10,6 +20,9 @@ function ease_in_out_quad(t) {
 }
 
 function animate_show(target) {
+    if (target === undefined) {
+        return
+    }
```

Comment:
I dont think this is the respoinsibility of this function to check if the arg was correct. Instead, check it on the call side, please.